README.md

```diff
@@ -35,6 +35,9 @@ make services-invoke-publish
 npm install --global yarn
 # Install the deps
 yarn
+# Install the deps for website-next
+cd website-next
+yarn
```

Comment:
We could also publish website-next and name it as a dependency, or just have it be ignored by Mocha.

Since we're intending to remove it anyway this works for now.